_posts/blog/2017-07-19-Actions-for-android-devs.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Google Assistant Development for Android Devs (Part 1)"
+author: Patrick Jackson
+categories: Development
+tags: [Google Assistant, Actions on Google, conversational interfaces, Android, Kotlin]
+---
+
+
+# Google Assistant Development for Android Devs (Part 1)
+
+![](https://developers.google.com/actions/images/surfaces-hero.svg)
+
+There is a lot of buzz and excitment around the [Google Assistant Actions](https://developers.google.com/actions/), and with good reason:  this is an entirely new platform for building experiences for a number of devices.  Building a conversational Action with Google Assistant means it will be available on Android, iOS, Google Home, and others as they com available.
+
+
+For the uninitiated, Actions on Google are third party "Actions" that can be built using Google's APIs.  Actions extend what the Google Assistant can do by letting user talk to them.  They can be accessed with trigger words such as 'Ok Google, talk to Ticketmaster', or 'Ok Google, ask ticketmaster find Yankee's games this weekend' (Ticketmaster Action coming soon!)
```

Comment:
- Can we use another word besides "Actions" to explain what they are? Bots? Assistants? Apps? Scripts?
  "...Actions on Google are third-party **apps** that can be built..."
- Need a hyphen: "...**third-party**..."
- Need a period at the end of the paragraph (after parenthetical or before, your choice)